lib/schema/json_shemata.ex

```diff
@@ -73,19 +86,23 @@ defmodule ApplicationRunner.JsonSchemata do
 
   def get_component_path(comp_type), do: "components/#{comp_type}.schema.json"
 
-  @impl true
-  def handle_call({:load_raw_schema, schema, component_name}, _from, schemata_map) do
-    {:reply, :ok, load_raw_schema(schema, schemata_map, component_name)}
-  end
+  # @impl true
+  # def handle_call({:load_raw_schema, schema, component_name}, _from, schemata_map) do
```

Comment:
Do we still need the `load_raw_schema` comments across application_runner ? If not, you can remove every comment that corresponds to the `load_raw_schema`